osu.Game/Graphics/Containers/Markdown/OsuMarkdownTextFlowContainer.cs

```diff
@@ -33,6 +39,27 @@ protected override void AddAutoLink(AutolinkInline autolinkInline)
         protected override SpriteText CreateEmphasisedSpriteText(bool bold, bool italic)
             => CreateSpriteText().With(t => t.Font = t.Font.With(weight: bold ? FontWeight.Bold : FontWeight.Regular, italics: italic));
 
+        protected override void AddCustomComponent(CustomContainerInline inline)
+        {
+            if (!(inline.FirstChild is LiteralInline literal))
+            {
+                base.AddCustomComponent(inline);
+                return;
+            }
+
+            string[] attributes = literal.Content.ToString().Trim(' ', '{', '}').Split();
+            string flagAttribute = attributes.SingleOrDefault(a => a.StartsWith(@"flag", StringComparison.Ordinal));
+
+            if (flagAttribute == null)
+            {
+                base.AddCustomComponent(inline);
+                return;
+            }
+
+            string flag = flagAttribute.Split('=').Last().Trim('"');
+            AddDrawable(new DrawableFlag(Enum.Parse<CountryCode>(flag)) { Size = new Vector2(20, 15) });
```

Comment:
Should probably use `TryParse` for safety and default to `CountryCode.Unknown` if failed? Since wiki is user edited a typo could crash the game.